react/PlaceOrder.tsx

```diff
@@ -107,11 +107,14 @@ const PlaceOrder: React.FC = () => {
                 return null
               }
 
+              const { merchantSellerPayments, ...payment } = transactionPayment
+
               return {
-                ...transactionPayment,
                 ...merchantPayment,
+                ...payment,
```

Comment:
Can you tell if there is any risk of overwriting?

Reply:
They have basically the same values, but I'll move back the payment object just to be sure